backend/app/assets/javascripts/spree/backend/admin.js

```diff
@@ -159,7 +159,6 @@ Spree.ready(function(){
       return ui;
   };
 
-  $('table.sortable').ready(function(){
```

Comment:
This is a change in the actual implementation and not necessary for Turbolinks to work, right? I think this should not be in this PR.